src/Resources/config/services.xml

```diff
@@ -26,6 +26,12 @@
             class="Sylius\PayPalPlugin\Generator\PayPalAuthAssertionGenerator"
         />
 
+        <service id="Sylius\PayPalPlugin\ApiPlatform\PayPalPayment">
+            <argument type="service" id="router" />
+            <argument type="service" id="Sylius\PayPalPlugin\Provider\AvailableCountriesProviderInterface" />
+            <tag name="api.payment" />
```

Comment:
Can you explain what does this tag do?

Reply:
this tag is used to collect all data providers from payment plugins, in Core we got PaymentConfiguration provider which allows us to send data to frontend based on picked payment method, f.eg if we pick paypal payment method we search throughout all payment methods with api.payment tag and see which one supports it, then we send payment data configuration to pay using specific payment method.